src/Discovery/DiscoveryDetails/Components/FieldGrouping/TabField/DataDownloadList/Utils/ProcessData.ts

```diff
@@ -1,16 +1,22 @@
-const ProcessData = (sourceFieldData:any) => {
+export const MAX_NUMBER_OF_ITEMS_IN_LIST = 200;
+
+export const ProcessData = (sourceFieldData:any) => {
   const dataWithOnlyTitlesOrFileNames = sourceFieldData[0].filter((item:any) => {
     if (!('title' in item || 'file_name' in item)) {
       console.debug('Item without title or file_name:', item);
     }
     return 'title' in item || 'file_name' in item;
   });
-  const processedDataForDataDownloadList = dataWithOnlyTitlesOrFileNames.map((obj:any) => ({
+  let processedDataForDataDownloadList = dataWithOnlyTitlesOrFileNames.map((obj:any) => ({
     title: obj.title || obj.file_name,
     description: obj.description,
     guid: obj.object_id,
   }));
-  return processedDataForDataDownloadList;
+  let dataForDataDownloadListHasBeenTruncated = false;
+  if (processedDataForDataDownloadList.length > MAX_NUMBER_OF_ITEMS_IN_LIST) {
+    processedDataForDataDownloadList = processedDataForDataDownloadList.slice(0, MAX_NUMBER_OF_ITEMS_IN_LIST);
+    dataForDataDownloadListHasBeenTruncated = true;
+  }
+  processedDataForDataDownloadList = processedDataForDataDownloadList.sort((a, b) => a.title.localeCompare(b.title, undefined, { numeric: true, sensitivity: 'base' }));
```

Comment:
The ticket HP-1978 has additional requirements. Are this requirements no longer needed or will they be addressed in a separate PR? 

```
File paths, if available, are added at the beginning of the filenames

Files at the root level (not inside any folder) will be displayed first, in alphabetical order

Files nested within folder(s), if applicable, will be displayed after those at the root level in alphabetical order